src/connector.js

```diff
@@ -861,13 +975,14 @@ function updateResultListState( newState ) {
 				else {
 					author = result.raw.author;
 				}
-				if( params.isContextSearch ) {
-					author = author.replace( ';', ', ' );
-				}
-				else {
-					author = author.replace( ',', ';' );
-					author = author.replace( ';' , '</li> <li>' );
-				}
+
+				author = author.replaceAll( ';' , '</li> <li>' );
```

Comment:
Can you please document this change in the description of your PR? Or open a separate PR with this change, since it is most likely unrelated to QS.